### DISCUSSION TITLE
Prisma not seed on Docker container (Windows platform)

### DISCUSSION THREAD

**Discussion by l2D** [prisma/prisma#7266] [Q&A] [answered]
Hello  @Prisma,

After I asked about seeding on #7199 . I got a new problem with seeding on windows. 
When I run ``` npx prisma migrate dev ``` and ```npx migrate reset ```, I got ``` Running seed from "prisma\seed.js" ... ``` 
but no response from seed files and the database has no update records. (either ```npx prisma db seed --preview-feature ```)

<pre><code>
Environment variables loaded from .env
prisma               : 2.23.0
@prisma/client       : 2.23.0
Current platform     : windows
Query Engine         : query-engine adf5e8cba3daf12d456d911d72b6e9418681b28b (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine     : migration-engine-cli adf5e8cba3daf12d456d911d72b6e9418681b28b (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine : introspection-core adf5e8cba3daf12d456d911d72b6e9418681b28b (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary        : prisma-fmt adf5e8cba3daf12d456d911d72b6e9418681b28b (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash : adf5e8cba3daf12d456d911d72b6e9418681b28b
Studio               : 0.393.0
Preview Features     : filterJson
</code></pre>

**Comment by janpio** [comment, marked as answer]
Can you please open an issue with the same content? Then we can try to reproduce this to see if this is a bug. Thanks.

**Reply by l2D** [reply]
> Can you please open an issue with the same content? Then we can try to reproduce this to see if this is a bug. Thanks.

Thank you, I created on #7268
